### PR TITLE
Logs: Adjust documentation for new log details

### LINKED ISSUE #61026: Logs: Adjust documentation for unified labels/detected-fields
We need to change [documentation](https://grafana.com/docs/grafana/latest/explore/logs-integration/) after https://github.com/grafana/grafana/issues/30221 is finished.

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

We changed how log details are displayed in https://github.com/grafana/grafana/pull/60448. This PR is updating the docs - mainly screenshots - accordingly.

Fixes #61026

### CODE REVIEW

**Review by gabor — APPROVED** [review]
LGTM, from the "content" perspective.

File: docs/sources/panels-visualizations/visualizations/logs/index.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ For logs where a **level** label is specified, we use the value of the label to
 
 Each log row has an extendable area with its labels and detected fields, for more robust interaction. Each field or label has a stats icon to display ad-hoc statistics in relation to all displayed logs.
 
-### Derived fields links
+### Data links
 
-By using Derived fields, you can turn any part of a log message into an internal or external link. The created link is visible as a button next to the Detected field in the Log details view.
+By using Data links, you can turn any part of a log message into an internal or external link. The created link is visible as a button in the Links section inside the Log details view.
```
Comment (chri2547) [inline docs/sources/panels-visualizations/visualizations/logs/index.md:37]:
```suggestion
By using data links, you can turn any part of a log message into an internal or external link. The created link is visible as a button in the **Links** section inside the **Log details** view.
```

Comment (svennergr) [reply docs/sources/panels-visualizations/visualizations/logs/index.md]: Thanks @chri2547. Applied in b06f329.

File: docs/sources/explore/logs-integration.md [hunk 2]
```diff
@@ -112,18 +112,16 @@ To automatically fix incorrectly escaped sequences that Explore has detected:
 
 Explore replaces these sequences. When it does so, the option will change from "Escape newlines" to "Remove escaping". Evaluate the changes as the parsing may not be accurate based on the input received. You can revert the replacements by clicking "Remove escaping".
 
-#### Derived fields links
+#### Data links
 
-By using Derived fields, you can turn any part of a log message into an internal or external link. The created link is visible as a button next to the Detected field in the Log details view.
-{{< figure src="/static/img/docs/explore/detected-fields-link-7-4.png" max-width="800px" caption="Detected fields link in Explore" >}}
+By using Data links, you can turn any part of a log message into an internal or external link. The created link is visible as a button in the Links section inside the Log details view.
```
Comment (chri2547) [inline docs/sources/explore/logs-integration.md:117]:
```suggestion
By using data links, you can turn any part of a log message into an internal or external link. The created link is visible as a button in the **Links** section inside the **Log details** view.
```

**Review by chri2547 — COMMENTED** [review]
A few suggestions. Thank you for the contribution!

**Review by chri2547 — APPROVED** [review]
LGTM! Thank you!

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**